### PR TITLE
ref: Improve log logic

### PR DESCRIPTION
**Pull request by kevinrenskers**
Follow-up of #2087.

#skip-changelog

### CODE REVIEW

**Comment by brustolin** [comment]
Please take a look at the UIEventTracker regarding the comment in the other PR.
I tried here and it is possible to not have a `currentActiveTransaction`

**Comment by kevinrenskers** [comment]
I think you need to refresh the PR? :)

File: Sources/Sentry/SentryCoreDataTracker.m [hunk 1]
```diff
@@ -30,15 +30,23 @@ - (NSArray *)managedObjectContext:(NSManagedObjectContext *)context
                                       description:[self descriptionFromRequest:request]];
     }];
 
+    NSArray *result = original(request, error);
```
Comment (brustolin) [inline Sources/Sentry/SentryCoreDataTracker.m:33]: The problem with this new solution is that you logging the beginning of the write process after it occurs. 
Both `started` and `finished` logs will occur basically at the same time.

Comment (kevinrenskers) [reply Sources/Sentry/SentryCoreDataTracker.m:33]: Good point

**Review by brustolin — APPROVED** [review]

**Pull request merged**